data/AyebakuroOruwori.json

```diff
@@ -0,0 +1,43 @@
+{
+  "avatar": "https://github.com/AyebakuroOruwori/hashNodeBlog/blob/main/66691177.jpeg",
+  "name": "Ayebakuro A. Oruwori",
+  "type": "personal",
+  "bio": "Software Engineer who works as a DevOps Engineer and Cloud Engineer at the moment.",
+  "links": [
+    {
+      "name": "Twitter",
+      "url": "https://twitter.com/Ayebakkuroo",
+      "icon": "FaTwitter"
+    },
+   
+   {
+      "name": "Instagram",
+      "url": "https://www.instagram.com/ayebakuro_o/",
+      "icon": "FaInstagram"
+    }, 
+
+  {
+      "name": "LinkedIn",
+      "url": "https://www.linkedin.com/in/ayebakuro-oruwori/",
+      "icon": "FaLinkedIn"
+    },
+
+  {
```

Comment:
You added Github twice, otherwise looks good!

Reply:
Thank you, I'll update it right away